Viper/Viper/HotelsModule/Presenter/HotelsPresenter.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  HotelsPresenter.swift
+//  Viper
+//
+//  Created by Алена Панченко on 13.02.2023.
+//
+
+import Foundation
+
+/// Презентер экрана отелей
+final class HotelsPresenter: HotelsPresenterProtocol {
+  
+    // MARK: Public Properties
+    
+    weak var view: HotelsViewInputProtocol?
+    var iterator: HotelsInteractorInputProtocol? {
+        didSet {
+            guard let index else { return }
+            iterator?.fetchHotels(index: index)
+        }
+    }
+    var router: HotelsRouterInputProtocol?
+    var index: Int?
+    var hotels: Hotels?
```

Comment:
```suggestion
    var hotels: Hotels?
```
у тебя иногда так, а иногда var hotels: [Hotels] = [] мне с опционалом больше нравится. Это не замечание